reference/UQ Handbook Chapter.ipynb

```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
```

Comment:
I would have thought so. I can easily add some MCMC background, but I wasn't clear how much of an MCMC tutorial it should be, as opposed to an intro to PyMC. I envisioned the target audience as being someone familiar with the methods, but not the software.